src/index.js

```diff
@@ -75,6 +75,10 @@ import { Registry } from './registry';
 import { StaveText } from './stavetext';
 import { GlyphNote } from './glyphnote';
 import { RepeatNote } from './repeatnote';
+import { TextFont } from './textFont';
+import { PetalumaScriptMetrics } from './fonts/petalumaScript_metrics';
+import { RobotoSlabMetrics } from './fonts/robotoSlab_metrics';
```

Comment:
So, looking at the content of these files, they look like glyphs, so should have the `_glyphs` suffix. The `_metrics` suffix is for customizing positioning/scale and other transformations. Also, let's find a way to separate the text glyphs from the music glyphs. Maybe `petalumascript_text_glyphs`?

(Sorry, I should've caught this in your previous PR.)

Reply:
Wouldn't the glyphs be in the actual font files, though?  I would say they are metrics, indexed by glyph, and I think advanceWidth/leftBearing would be regarded by many as (a subset of) font metrics.  But I will call them glyphs if you like :)

Reply:
Yes, you're right, however the `x_min`, `x_max`, `ha` etc. fields are typically part of the glyphs (because they capture bounding box info.) If you look at the other `_glyph` files, you'll see the actual glyphs and the same bounding box information.

Anyhow, since there are no glyphs here, maybe it's okay to call them metrics. Can you call them `_textmetrics` instead? Also do you have the code to generate these metrics?